Tasks/SqlAzureDacpacDeployment/task.json

```diff
@@ -151,8 +147,15 @@
 			"defaultValue": "true",
 			"helpMarkDown": "If selected, deletes the IP Address exception in the Azure SQL DB Firewall after the task ends"
 		}
-	],
-	"sourceDefinitions": [],
+	],	
+    "sourceDefinitions": [ 
+     { 
+         "target": "IpDetectionMethod", 
```

Comment:
This is good change. It is good that service ie telling what it supports. Right way to model client driven by service contract.